src/core/handlers/RequestHandler.ts

```diff
@@ -216,6 +217,7 @@ export abstract class RequestHandler<
   public async run(args: {
     request: StrictRequest<any>
     resolutionContext?: ResponseResolutionContext
+    requestId?: string
```

Comment:
should this be required instead of optional?  are we leaving it optional to avoid making it required for non-standard usage?  just curious if we can accidentally miss this if it's optional?

Reply:
I'm wondering if this is considered exposed of it it's OK to change this function signature to get the required arg before the optional one

So from this
```ts
/**
 * Returns a mocked response for a given request using following request handlers.
 */
export const getResponse = async <Handler extends Array<RequestHandler>>(
  request: Request,
  handlers: Handler,
  resolutionContext?: ResponseResolutionContext,
  requestId?: string,
): Promise<ResponseLookupResult | null> => {
```
to this
```ts
/**
 * Returns a mocked response for a given request using following request handlers.
 */
export const getResponse = async <Handler extends Array<RequestHandler>>(
  request: Request,
  handlers: Handler,
  requestId: string,
  resolutionContext?: ResponseResolutionContext,
): Promise<ResponseLookupResult | null> => {
```

Reply:
@skvale, I believe your last question is resolved since the `run()` method now accepts a single object argument where the argument position is irrelevant. Thanks for updating the PR!